.github/workflows/integration-tests.yml

```diff
@@ -97,4 +97,27 @@ jobs:
           node-version: 12.x
       - run: npx lerna bootstrap
       - run: npx lerna run compile
-      - run: npx lerna run integration/local --stream
\ No newline at end of file
+      - run: npx lerna run integration/local --stream
+
+  kubernetes-deploy-integration-tests:
+    name: Kubernetes deployment integration tests
+    runs-on: ubuntu-latest
```

Comment:
I wonder if it wouldn't be better to fix the ubuntu version to avoid running into breaking changes on the newest versions 🤔 I guess is a trade-off; either ensuring test pass always running in the same environment or have them always up to date fixing those breaking changes. So I'll leave it up to you

Reply:
Good point! For now I think it's a nice "problem" to have since Ubuntu releases a version every 2 years, we can create a very small PR changing all versions for all integration tests or just make it compatible with the next one :v:.